### PR TITLE
Amp 870 renaming dashboard result

### PR DESCRIPTION
**Pull request by dfische3**
Renaming dashboard to WorkflowResult.

### CODE REVIEW

File: src/main/java/edu/indiana/dlib/amppd/controller/MediaController.java [hunk 1]
```diff
@@ -58,14 +51,14 @@
...
-	 * @param id ID of the given dashboardResult
+	 * Serve the output file of the given workflowResult by redirecting the request to the AMPPD UI Apache server.
+	 * @param id ID of the given workflowResult
 	 * @return the content of the output file
 	 */
-	@GetMapping("/dashboard/{id}/output")
-	public ResponseEntity<Object> serveDashboardOutput(@PathVariable("id") Long id) {		
-    	log.info("Serving output for dashboardResult ID " + id);
-    	String url = mediaService.getDashboardOutputSymlinkUrl(id);
+	@GetMapping("/workflow-result/{id}/output")
```
Comment (yingfeng-iu) [inline src/main/java/edu/indiana/dlib/amppd/controller/MediaController.java:58]: below workflow-results/ is used for the path, lets use the plural here too to be consistent

File: src/main/java/edu/indiana/dlib/amppd/controller/WorkflowResultController.java [hunk 2]
```diff
@@ -8,29 +8,29 @@
...
+		log.info("Received request inside getWorkflowResults");
+		return workflowResultsService.getWorkflowResults(query);
 	}
 	
-	@PostMapping(path = "/dashboard/isfinal/{id}", consumes = "application/json", produces = "application/json")
+	@PostMapping(path = "/workflow-results/isfinal/{id}", consumes = "application/json", produces = "application/json")
 	public boolean setIsFinal(@PathVariable("id") Long id, @RequestParam("isFinal") boolean isFinal){
 		log.info("Setting file to final: " + id);
-		return dashboardService.setResultIsFinal(id, isFinal);
+		return workflowResultsService.setResultIsFinal(id, isFinal);
```
Comment (yingfeng-iu) [inline src/main/java/edu/indiana/dlib/amppd/controller/WorkflowResultController.java:33]: should be workflowResultService?

File: src/main/java/edu/indiana/dlib/amppd/controller/WorkflowResultController.java [hunk 3]
```diff
@@ -44,26 +44,26 @@ public boolean setIsFinal(@PathVariable("id") Long id, @RequestParam("isFinal")
 	 * parimaryfile/bundle to add the results.
 	 */	
 	/**
-	 * Refreshes the whole DashboardResults table iteratively by retrieving and processing workflow invocations per primaryfile,
+	 * Refreshes the whole WorkflowResults table iteratively by retrieving and processing workflow invocations per primaryfile,
 	 * unless the lumpsum mode is specified and true, in which case, retrieve and process all workflow invocations at once.
 	 * It's recommended to turn lumpsum off if request to Galaxy tends to timeout due to large amount of records.
-	 * The DashboardResults table is typically refreshed for the following cases:
+	 * The WorkflowResults table is typically refreshed for the following cases:
```
Comment (yingfeng-iu) [inline src/main/java/edu/indiana/dlib/amppd/controller/WorkflowResultController.java:50]: WorkflowResult table

**Review by yingfeng-iu — CHANGES_REQUESTED** [review]
Looks good. I made a few suggestions for consistency. Also, if you haven't done this, would be a good idea to run the ignored tests locally (except the known failed BagServiceTests)  to make sure we didn't break anything.

File: src/main/java/edu/indiana/dlib/amppd/controller/WorkflowResultController.java (hunk 3 above)
Comment (dfische3) [reply src/main/java/edu/indiana/dlib/amppd/controller/WorkflowResultController.java]: Fixed!

File: src/main/java/edu/indiana/dlib/amppd/controller/WorkflowResultController.java (hunk 2 above)
Comment (dfische3) [reply src/main/java/edu/indiana/dlib/amppd/controller/WorkflowResultController.java]: Changed!

File: src/main/java/edu/indiana/dlib/amppd/controller/MediaController.java (hunk 1 above)
Comment (dfische3) [reply src/main/java/edu/indiana/dlib/amppd/controller/MediaController.java]: Good catch!

**Comment by dfische3** [comment]
I have updated the suggestions.  I will uncomment and run all of the test before merging.  Thanks!
 <http://www.portalmedia.com/>	Dan Fischer  |  Senior Developer
PortalMedia, Inc. <http://www.portalmedia.com/>      tel:  608.509.4198 <tel:+16085094198>

**Review by yingfeng-iu — APPROVED** [review]
Thanks!

**Pull request merged**